onnxruntime/core/providers/webgpu/tensor/concat.cc

```diff
@@ -38,48 +38,6 @@ WEBGPU_CONCAT_VERSIONED_KERNEL(4, 10)
 WEBGPU_CONCAT_VERSIONED_KERNEL(11, 12)
 WEBGPU_CONCAT_KERNEL(13)
 
-const std::string AppendCalCulateInputIndexFunction(size_t input_count) {
```

Comment:
Code from these functions are moved into the GenerateShaderCode function. Modularity is getting lost. Is this changes necessary? We could still keep these AppenXyzFunction function and use the return value to insert into additional implementation.